support-frontend/assets/components/personalDetails/stateSelect.tsx

```diff
@@ -49,7 +49,7 @@ export function StateSelect({
 				name={'billing-state'}
 			>
 				<>
-					<Option value="">
+					<Option value="" selected>
```

Comment:
Isn't the first option selected by default?

Reply:
Left this in whilst playing with it, removed.